translations/russian/research/notes.md

```diff
@@ -0,0 +1,7 @@
+## Using russian in web apps
+
+* Most of web apps uses more formal, neutral words.
+* Apps use infinitive for common words.
+* After switching languages interface doesn'n change.
+* Russian language is left-to-rigth.
+* Russian uses cyrilic symbols.
```

Comment:
Could you please elaborate a bit more on each of these points and give examples where you can? I think that would be very useful! :)